perception/traffic_light_fine_detector/src/nodelet.cpp

```diff
@@ -270,13 +270,17 @@ void TrafficLightFineDetectorNodelet::detectionMatch(
   }
 
   out_rois.rois.clear();
-  for (const auto & p : bestDetections) {
+  for (const auto & r : id2expectRoi) {
```

Comment:
[IMO] How about the following update? It is easier to understand rather than `r` or `p`.

```cpp
for (const auto & [tlr_id, _] : id2expectRoi) {
  TrafficLightRoi tlr;
  tlr.traffic_light_id = tlr_id;
  if (bestDetections.count(tlr_id) {
    const auto & object = bestDetections.at(tlr_id);
    tlr.roi.x_offset = object.x_offset;
    tlr.roi.y_offset = object.y_offset;
    tlr.roi.width = object.width;
    tlr.roi.height = object.height;
  }
}
```

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/5934/commits/c289d20bae7e9ba627491602ed2d659deb918d7e.